### ISSUE TITLE
Item list with value and text 

### ISSUE THREAD

**Issue by cristo1985** [LeaVerou/awesomplete#16849]
Hi,
is there a trick to create list with value and text with this best i saw plugin?
Thanks
Cris

**Comment by LeaVerou** [comment]
I'm sorry, I don't understand what you're saying. Could you rephrase? Maybe use google translate?

**Comment by cristo1985** [comment]
Hi,
if i receive json data created with this mine class:

public class CustomerData {

public int idCustomer;
public string name;
public string surname;

} 

And i wanna make a textbox with awesomplete function but not dropdownlist bus just simply a text box...with this made <option value="idCustomer"> surname + " " + name </option> and when i click on list appeared (created with awesomplete) get the value of selected item and make call a function passing idCustomer.
Thanks
Cris

**Comment by vlazar** [comment]
Separate `label/value` is landed #16866
It is also possible to initialize list with separate `label/value` via `<datalist>` or `<ul>`.

Read about different `label` and `value` at the end of [Basic usage](http://leaverou.github.io/awesomplete/#basic-usage) section and about new `data` method at the end of [Extend](http://leaverou.github.io/awesomplete/#extensibility) section.
